### PR TITLE
Release v0.5.0 -- orbits and stand-alone HyP3 job support

### PR DESCRIPTION
**Pull request by jhkennedy**

### CODE REVIEW

**Review by cmarshak — DISMISSED** [review]
I tested the throughput of the orbit credential fixes and am confident this will be a good stop cap until `sentineleof` is fixed more permanently.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Latest Changes
```
Comment (jhkennedy) [inline CHANGELOG.md:9]:
```suggestion
## [0.5.0]
```
Need to change this

**Review by jhkennedy — COMMENTED** [review]
@cmarshak what's going on with the tests?

**Comment by cmarshak** [comment]
I am mystified. The circleCI outputs are confusing, but they are related to the mocks of the hyp3 metadata e.g. 

```
 def test_GUNW_hyp3_metadata_update_different_prefix(test_gunw_json_path, test_gunw_json_schema_path, tmp_path, mocker):
        """This test performs the GUNW entrypoint using a different input bucket/prefix than the output bucket/prefix.
        Only updates the json. Monkey patches the upload/download to/from s3 and the actual computation.
        """
        temp_json_path = tmp_path / 'temp.json'
        shutil.copy(test_gunw_json_path, temp_json_path)
    
        # We only need to make sure the json file is passed, the netcdf file name will not have
        # any impact on subsequent testing
        mocker.patch("RAiDER.aws.get_s3_file", side_effect=['foo.nc', temp_json_path])
        mocker.patch("RAiDER.aws.upload_file_to_s3")
        mocker.patch("RAiDER.aria.prepFromGUNW.main", return_value=['my_path_cfg', 'my_wavelength'])
        mocker.patch('RAiDER.aria.prepFromGUNW.check_hrrr_dataset_availablity_for_s1_azimuth_time_interpolation',
                     side_effect=[True])
        mocker.patch("RAiDER.aria.prepFromGUNW.check_weather_model_availability", return_value=True)
        mocker.patch("RAiDER.cli.raider.calcDelays", return_value=['file1', 'file2'])
        mocker.patch("RAiDER.aria.calcGUNW.tropo_gunw_slc")
    
        iargs = ['--weather-model', 'HRES',
                 '--bucket', 'myBucket',
                 '--bucket-prefix', 'myOutputPrefix',
                 '--input-bucket-prefix', 'myInputPrefix',]
>       calcDelaysGUNW(iargs)
```

But #640 succeeded - not sure. Maybe let's retry?

**Comment by cmarshak** [comment]
So with your work on #642 fixed the test and this release, which is now ready to support the orbit issue AND ongoing work with submitting individual hyp3 job ids.

**Comment by jhkennedy** [comment]
@cmarshak correct. The Changelog just needs to be updated and this should be ready to go. Would be nice to get this out today, but I'm on baby duty today and tomorrow so won't be able to help

**Review by bbuzz31 — APPROVED** [review]

**Pull request merged**